setup.py

```diff
@@ -37,7 +37,7 @@
         "testing": [
             "responses>=0.12,<1",
             "Pillow>=4.0.0,<9.0.0",
-            "wagtailmedia>=0.6",
+            "wagtailmedia>=0.6,<0.13",
```

Comment:
wagtailmedia >= 0.13 has and issue raised that is the cause of a testing failure here: https://github.com/torchbox/wagtail-webstories/actions/runs/4313681735/jobs/7525779129

I've pinned it here so it's more obvious it needs attention for wagtail >= 5.0